vault/logical_system_paths.go

```diff
@@ -537,6 +867,7 @@ func (b *SystemBackend) capabilitiesPaths() []*framework.Path {
 				"path": &framework.FieldSchema{
 					Type:        framework.TypeCommaStringSlice,
 					Description: "(DEPRECATED) Path on which capabilities are being queried. Use 'paths' instead.",
+					Deprecated:  true,
```

Comment:
Callbacks at L878 could have been missed out.

Reply:
The paths were converted on an as-needed basis by looking at the output and docs. Reasons for converting/adding:

- No documentation present at all (common for unauth'd methods)
- HelpSynopsis is not appropriate for all operations. If something supports most CRUD ops, they should each read differently. But in many cases there is just one operation and the help describes it well, so there was no immediate need to change it.

There are a lot of paths to convert, so ATM I just tackled ones that visibly improved output docs.